WordPress/Classes/ViewRelated/Blog/DiscussionSettingsViewController.swift

```diff
@@ -127,40 +127,26 @@ public class DiscussionSettingsViewController : UITableViewController
         return cell
     }
 
-    public override func tableView(tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        guard let headerText = sections[section].headerText else {
-            return CGFloat.min
-        }
-
-        return WPTableViewSectionHeaderFooterView.heightForHeader(headerText, width: tableView.bounds.width)
-    }
-
-    public override func tableView(tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+    public override func tableView(tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
         guard let headerText = sections[section].headerText else {
```

Comment:
For this and the footer title, you can just `return sections[section].headerText` directly – if it doesn't exist, it'll be nil anyway.